ports/windows/README.md

```diff
@@ -29,13 +32,37 @@ Install following packages using cygwin's setup.exe:
 
 Build using:
 
+    make
+or
     make CROSS_COMPILE=i686-w64-mingw32-
 
 Or for 64bit:
 
     make CROSS_COMPILE=x86_64-w64-mingw32-
 
 
+Building using MSYS2 / Mingw on Windows
+---------------------------------------
+
+Install MSYS2 64 bit from msys2.org
+Run the installer msys2-x86_64-xxxxxxxx.exe
```

Comment:
Note that just doewnload the base.tar.gz or whatever it's compressed like also works.

Reply:
I am trying to provide simple instructions for newbies by documenting a method that is tested and easy. I'm not sure that a 12 year old that's just started programming with  a micro:bit and a Windows laptop would even know what  a tar.gz is. Likewise a self taught Python programmer that only has Windows experience. I went through a lot of pain because I followed instructions on line that had me install a toolchain that included i686-w64-mingw32-gcc but not gcc. So I think its better to document a simple method that works. If experience people want to do it differently they can at their own risk.

Reply:
Ok. It's just that I once had problems with the installer, don't recall what exactly, so since then I just used the compressed version. But indeed for novices that is one extra possibly hard step.